src/PubSubClient.h

```diff
@@ -88,7 +90,9 @@ class PubSubClient {
    unsigned long lastOutActivity;
    unsigned long lastInActivity;
    bool pingOutstanding;
-   MQTT_CALLBACK_SIGNATURE;
+   MQTT_CALLBACK_SIGNATURE;  
+   MQTT_CALLBACK_SIGNATURE_WITH_ARG; 
+   void* arg;
```

Comment:
This should have a more descriptive name - `arg` is too generic to know what its related to.